sound/soc/sof/topology.c

```diff
@@ -790,7 +792,8 @@ static int sof_parse_string_tokens(struct snd_soc_component *scomp,
 				   void *object,
 				   const struct sof_topology_token *tokens,
 				   int count,
-				   struct snd_soc_tplg_vendor_array *array)
+				   struct snd_soc_tplg_vendor_array *array,
+				   int offset)
```

Comment:
wouldn't `size_t` be a better type for `offset` everywhere?

Reply:
hmm yes, let me try to fix those